java/configuration/src/main/java/sleeper/configuration/properties/instance/CdkDefinedInstanceProperty.java

```diff
@@ -124,7 +124,22 @@ public interface CdkDefinedInstanceProperty extends InstanceProperty {
             .description("The name of the table responsible for tracking query progress.")
             .propertyGroup(InstancePropertyGroup.QUERY)
             .build();
-
+    CdkDefinedInstanceProperty LEAF_PARTITION_QUERY_QUEUE_URL = Index.propertyBuilder("sleeper.leaf.partition.query.queue.url")
```

Comment:
These properties should start with sleeper.query to be consistent with the other properties.

Reply:
Updated